wiki/introduction/usage.html

```diff
@@ -9,13 +9,13 @@
 <hr>
 
 <h2 id="toc0">General Workflow</h2>
-<p>Here's how you'll generally be downloading things with SABnzbd.</p>
-<ul>
-    <li>Search for something you want to download at your <a href="/wiki/introduction/nzb-sources">usenet indexer</a> of choice.</li>
-    <li>Download an NZB describing that data</li>
-    <li>Give that NZB to SABnzbd in one of a wide variety of ways</li>
+<p>Here's how you'll generally download things with SABnzbd:</p>
+<ol>
```

Comment:
Changed this to an ordered list as this block describes a process that should be completed in order.